Content.Shared/_RMC14/Xenonids/Evolution/XenoDevolveUI.cs

```diff
@@ -14,3 +14,9 @@ public sealed class XenoDevolveBuiMsg(EntProtoId choice) : BoundUserInterfaceMes
 {
     public readonly EntProtoId Choice = choice;
 }
+
+[Serializable, NetSerializable]
+public sealed class XenoDeStrainBuiMsg(EntProtoId choice) : BoundUserInterfaceMessage
```

Comment:
You don't need to pass in the choice, the server can just get the component and which id its supposed to destrain into